src/modules/hostname/src/lib/HostNameBase.cpp

```diff
@@ -87,7 +87,7 @@ int HostNameBase::Set(
 
     if (!IsValidObjectName(objectName, true))
     {
-        OsConfigLogError(HostNameLog::Get(), ERROR_INVALID_OBJECT, "Set", !objectName ? "-" : objectName, g_propertyName, g_propertyHosts);
+        OsConfigLogError(HostNameLog::Get(), ERROR_INVALID_OBJECT, "Set", !objectName ? "-" : objectName, g_propertyDesiredName, g_propertyDesiredHosts);
```

Comment:
>!objectName ? "-" : objectName [](http://example.com/codeflow?start=74&length=30)

nit: why not reversed, would look simpler overall

Reply:
objectName ? objectName : "-"

Reply:
Fixed, sorry for the delay.